<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_window</string>
  </property>
  <widget class="QLabel" name="label_height">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_category">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_professional">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Professional:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_results">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>180</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Results:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="info_height">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>100.000000000000000</double>
   </property>
   <property name="maximum">
    <double>300.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="info_category">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="info_professional">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>92</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="info_name">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_spr_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Sportsman #1</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_spr_1">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>66</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_spr_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Sportsman #2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_spr_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Sportsman #3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_spr_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Sportsman #4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_spr_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Sportsman #5</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_spr_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>66</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_spr_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>66</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_spr_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>66</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_spr_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>410</y>
     <width>66</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_age">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>180</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="info_age">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>223</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="info_res1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="info_res2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="info_res3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_empty">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
